<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>627</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Annotation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_4" columnstretch="7,0">
      <item row="0" column="1">
       <layout class="QGridLayout" name="left_panel" rowstretch="1,0,0,0,0,0,0">
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="FFT_layout"/>
        </item>
        <item row="6" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="Accept">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Accept</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="PreviousButton">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>&lt; Previous</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="NextButton">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Next &gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <widget class="QComboBox" name="EventDropdown_Box">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>--- Event Type ---</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Spike</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Real</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Artifact</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="FFT_layout_2"/>
        </item>
        <item row="4" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Event Information</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Channel:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="channel_name_textbox">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(235, 235, 235);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Event Length:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="length_textbox">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(235, 235, 235);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Start</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="start_textbox">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(235, 235, 235);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>End</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="end_textbox">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(235, 235, 235);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Model Prediction:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="model_textbox">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(235, 235, 235);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_13"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="AnotationDropdownBox"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="VisulaizationVerticalLayout"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1116</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionStart_Over"/>
   </widget>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStart_Over">
   <property name="text">
    <string>Reset Annotation</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
